.github/workflows/ci.yaml

```diff
@@ -156,13 +160,23 @@ jobs:
           # in PR synch the latest commit of the branch is found in github.event.pull_request.head.sha instead
 
           if [ "${{ github.event_name }}" = "release" ] && [ "${{ github.event.action }}" = "released" ]; then
-            echo "::set-output name=tag::${REF_ID/refs-tags-/}"
+            COMMIT_SHA=""
+            IMAGE_TAG="${REF_ID/refs-tags-/}"
           elif [ "${{ github.event_name }}" = "pull_request" ]; then
-            echo "::set-output name=tag::$REF_ID-${{ github.event.pull_request.head.sha }}"
+            COMMIT_SHA="${{ github.event.pull_request.head.sha }}"
+            IMAGE_TAG="$REF_ID-${{ github.event.pull_request.head.sha }}"
           else
-            echo "::set-output name=tag::$REF_ID-$GITHUB_SHA"
+            COMMIT_SHA=$GITHUB_SHA
+            IMAGE_TAG="$REF_ID-$GITHUB_SHA"
           fi
 
+          echo "::set-output name=commit-sha::$COMMIT_SHA"
+
+          # IMAGE_TAG is used in the Build Docker Image step.
+          # We can easily build the image-tag from REF_ID and COMMIT_SHA for non-releases in the step inputs
```

Comment:
all of the builds are actually releases, either to `production`, `staging-default` or just PRs. Could you specify that we cannot do that for `production` releases.

Reply:
added